database/migrations/2016_03_22_124953_rename_phone_number_column_on_groups.php

```diff
@@ -12,7 +12,7 @@ class RenamePhoneNumberColumnOnGroups extends Migration
      */
     public function up()
     {
-        DB::statement('ALTER TABLE `groups` CHANGE phone_number phone tinyint(1)');
```

Comment:
This is better, but not the "laravel-y" way to do this.  See https://laravel.com/docs/5.0/schema#changing-columns